### DISCUSSION TITLE
Group accessibility issues / Page rule

### DISCUSSION THREAD

**Discussion by vv2017kl** [requarks/wiki#6624] [Error / Bug Report]
I am observing that any rule with "Page starts with" , if it has a single folder path, the system works fine. But if we include a subfolder in the path, then the group rule does not work.
![image](https://github.com/requarks/wiki/assets/128370503/5a81fe6c-2d11-4ebd-9392-2b05551ab07a)

**Comment by NGPixel** [comment]
Do you have other rules that could override this one? Have you tried in lowercase?

**Reply by vv2017kl** [reply]
yes, removed all other rules and tested with just one rule with subfolder path. Whenever subfolder is in the path , it does not work. Have tried in lowercase also

**Comment by vv2017kl** [comment]
Kindly confirm at the earliest if this is a bug and/or if there is any workaround.

**Reply by NGPixel** [reply]
I can't replicate your issue.

**Comment by P4mes4n** [comment]
You need to be more specific. What do you mean by the rule doesn't work?

**Comment by vv2017kl** [comment]
steps to replicate
- Page structure   
- customers, customers/customer1, customers/customer2

group 1 : customer1accessgroup, path starts with en/customers/customer1
group 2 : customer2accessgroup, path starts with en/customers/customer2
group3 : accessallcustomers, path starts with en/customers

users with group1 or even group 2 cannot see their customer1 or customer2 respectively [[[ I guess this is the defect ]]]]
user with group 3 can see all customers

**Reply by NGPixel** [reply]
You're including the locale code in your paths which is wrong. Remove the `en/` part.

**Reply by P4mes4n** [reply]
This is a very basic structure and should not give any issues.
I just did a quick test with this setup and have the expected result.

So you either have a very strange bug in your setup (which I think is unlikely) or there is a misunderstanding or misconfiguration.
You are aware group1 and group2 have no rights to see `/customers` so you need to access `/customers/customer1(2)` directly?

**Reply by vv2017kl** [reply]
That is correct; when users login who belong to customer1 group, then should see anything after / customers/customer1, and same for customer2 group users to see customer2 pages. And when group3 users login, they can see all customers. Tried multiple times, will try fresh again. Maybe since users are associated with many other groups also, will remove and review/revert again.

**Reply by NGPixel** [reply]
You clearly have other groups with deny rules that are interfering with this group.

**Comment by vv2017kl** [comment]
That was just for explanation. I am not including in locale.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
